dns/cloudflare.go

```diff
@@ -106,10 +108,18 @@ func (cf *Cloudflare) addUpdateDomainRecords(recordType string) {
 			return
 		}
 
-		// 存在参数才进行筛选
-		comment := domain.GetCustomParams().Get("comment")
-		if comment != "" {
-			comment = fmt.Sprintf("&comment=%s", comment)
+		// The name of DNS records in Cloudflare API expects Punycode.
+		//
+		// See: cloudflare/cloudflare-go#688
```

Comment:
what does `cloudflare/cloudflare-go#688` mean?

Reply:
It says this:

> Cloudflare API expects ASCII forms (e.g., `xn--138h.org`) for DNS record names.

This is same in our case.

Reply:
I can't find the `cloudflare/cloudflare-go#688` code

Reply:
> I can't find the `cloudflare/cloudflare-go#688` code

That's because I linked to the issue and not PR, fixed.

Reply:
okay